packages/ra-ui-materialui/src/layout/Error.tsx

```diff
@@ -52,7 +52,15 @@ function goBack() {
     window.history.go(-1);
 }
 
-const Error = props => {
+export interface ErrorComponentProps {
```

Comment:
ErrocComponentProps => ErrorProps (we never include the name "component" in other interface names)